### PR TITLE
Add rules around Rallying

### LINKED ISSUE #14: What do we call passing the turn as a Lad in order to avoid paying the Uprising flip cost?
Needs a cool name. Any ideas?

### PR DESCRIPTION
**Pull request by seanKenkeremath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,12 +134,21 @@ As long as there is an active Uprising nail and the current player is a Lad, the
...
 
+#### Rallying the Lads
+
+If there is an active Uprising nail, a Lad may choose to call "Rally". If they do, their turn ends and they forfeit the ability to perform flips on that turn. By callling "Rally", that Lad has effectively contributed 1 flip to the next Lad who calls "Uprising" and performs an Uprising flip. For instance, if a Lad rallies and the next Lad in turn order calls "Uprising", that Lad can now perform a single flip and strike the nail (where they would normally need at least 2 flips). Factoring in Rallies, the total number of strikes allowed to a Lad during an Uprising turn becomes `f - 1 + r` where `f` is the number of flips
```
Comment (austenlux) [inline README.md]: says `callling` instead of `calling`

Comment (austenlux) [reply README.md]: Also, maybe worth calling out that `r` in the equation is the number of Rallies? Just like it's specified that `f` is the number of flips. Probably obvious already, either way.

**Review by austenlux — CHANGES_REQUESTED** [review]
See comments

File: README.md (hunk 1 above)
Comment (seanKenkeremath) [reply README.md]: oh yeah i meant to do that. good call

Comment (seanKenkeremath) [reply README.md]: @austenlux Fixed

**Review by austenlux — APPROVED** [review]

**Pull request merged**